va/va_test.go

```diff
@@ -142,14 +141,11 @@ func setup(srv *httptest.Server, maxRemoteFailures int, userAgent string, remote
 	if err != nil {
 		panic(fmt.Sprintf("Failed to create validation authority: %v", err))
 	}
-	if remoteVAs != nil {
-		va.remoteVAs = remoteVAs
-	}
 	return va, logger
 }
 
 func setupRemote(srv *httptest.Server, userAgent string, mockDNSClientOverride bdns.Client, perspective, rir string) (RemoteClients, *blog.Mock) {
```

Comment:
To resolve the lint finding (also requires changes elsewhere):
```suggestion
func setupRemote(srv *httptest.Server, userAgent string, mockDNSClientOverride bdns.Client, perspective, rir string) RemoteClients {
```